src/components/catalogue/CatalogueHeader.js

```diff
@@ -53,7 +53,9 @@ const CatalogueHeader = ({
             >
               {sources.map((source, index) => (
                 <MenuItem value={index} key={source.id}>
-                  {source.name}
+                  {(source.lang != null
```

Comment:
Switching to the v3 sources endpoint causes all the sources to be enabled and the catalogue was confusing since it looked like there were several copies of each source even though it was just the same source in different languages. I've added this language indicator to workaround this problem for now but it would be better if we had a screen to enable/disable sources.